docs/fundamentals/platform.rst

```diff
@@ -1,2 +1,279 @@
+Bus flux_line_q0_bus:  -----|QCM1|------|qubit_0|----
+Bus drive_line_q1_bus:  -----|QCM-RF1|------|qubit_1|----
+Bus flux_line_q1_bus:  -----|QCM1|------|qubit_1|----
+
+which displays the connections between the buses, instruments and elements of the chip.
+
+|
+
+Connecting and setting up the instruments with Platform:
+---------------------------------------------------------
+
+After building the platform, you need to connect to the instruments, set all the parameters defined in the runcard, and turn on the sources outputs using the following methods:
+
+.. code-block:: python
+
+    platform.connect()
+    # Connects to all the instruments and blocks the connection for other users.
+    # You must be connected to proceed with the following steps.
+
+    platform.initial_setup()
+    # Sets the values of the runcard (serialized platform) to the connected instruments.
+    # You might want to skip this step if you think no parameters have been modified since last time, but we recommend doing it anyway.
+
+    platform.turn_on_instruments()
+    # Turns on the signal output for the generator instruments (RF, voltage sources and current sources).
+    # This does not actually turn on the instruments of the laboratory, it only opens the signal output generation of the sources.
+    # You might want to skip this step if the instruments outputs are already open, but again, we recommend doing it anyway.
+
+.. note::
+
+    To connect, your computer must be in the same network of the instruments specified in the runcard (with their IP's addresses).
+
+|
+
+Executing a circuit with Platform:
+-----------------------------------
+The Platform offers the capability to execute circuits defined with `Qibo <https://qibo.science/>`_, an open-source middleware for quantum computing.
+
+To execute a circuit, you first need to build, connect, and set up the platform as shown in the above examples. Then, define your
+Qibo circuit, for example, a pi pulse and a measurement gate on qubit ``q`` (``int``):
+
+.. code-block:: python3
+
+    from qibo.models import Circuit
+    from qibo import gates
+
+    circuit = Circuit(q+1)
+    circuit.add(gates.X(q))
+    circuit.add(gates.M(q))
+
+And you are ready to execute the circuit with the platform:
+
+>>> result = platform.execute(program=circuit, num_avg=1000, repetition_duration=6000)
+>>> result.array
+array([[5.],
+        [5.]])
+TODO: !!! Change this results for the actual ones !!!
+
+getting the integrated values of the I/Q signals received by the digitizer!
+
+.. note::
+
+    When disabling scope acquisition mode, the array obtained has shape `(#sequencers, 2, #bins)`. In this case,
+    given that you are using only 1 sequencer to acquire the results, you would obtain an array with shape `(2, #bins)`.
+
+|
+
+Running a Rabi sequence with Platform:
+---------------------------------------
+
+To perform a Rabi sequence, build, connect and set up the platform, and then create a circuit with a
```

Comment:
the name Rabi _sequence_ feels off to me, perhaps Rabi _experiment_ or Rabi _sweep_ would be better? But maybe it's just me

Reply:
I used the nomenclature I found in a paper! But I actually like more rabi sweep, changing it!